project/plugins.sbt

```diff
@@ -3,11 +3,11 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefact
   Resolver.ivyStylePatterns
 )
 
-addSbtPlugin("uk.gov.hmrc"         % "sbt-auto-build"           % "3.20.0")
+addSbtPlugin("uk.gov.hmrc"         % "sbt-auto-build"           % "3.22.0")
 addSbtPlugin("uk.gov.hmrc"         % "sbt-distributables"       % "2.5.0")
-addSbtPlugin("org.playframework"   % "sbt-plugin"               % "3.0.1")
+addSbtPlugin("org.playframework"   % "sbt-plugin"               % "3.0.3")
 addSbtPlugin("org.scalastyle"      % "scalastyle-sbt-plugin"    % "1.0.0" exclude ("org.scala-lang.modules", "scala-xml_2.12"))
-addSbtPlugin("org.scoverage"       % "sbt-scoverage"            % "2.0.9")
+addSbtPlugin("org.scoverage"       % "sbt-scoverage"            % "2.0.12")
 addSbtPlugin("org.scalameta"       % "sbt-scalafmt"             % "2.5.2")
 addSbtPlugin("io.github.irundaia"  % "sbt-sassify"              % "1.5.2")
 addSbtPlugin("net.ground5hark.sbt" % "sbt-concat"               % "0.2.0")
```

Comment:
Please update sbt-concat to the [latest](https://github.com/sbt/sbt-concat?tab=readme-ov-file#plugin) (`net.ground5hark.sbt` has changed to `com.github.sbt`), thanks

Reply:
sbt-accessibility-linter can also be bumped to [1.0.0](https://github.com/hmrc/sbt-accessibility-linter/releases).